Command/AbstractGenerateCommand.php

```diff
@@ -59,4 +60,17 @@ protected function getTotalCount(array $config)
 
         return $totalCount;
     }
+
+    /**
+     * Create the output dir where fixtures will be generated if necessary.
+     *
+     * @param string $outputDir
+     */
+    protected function createOutputDirIfNecessary($outputDir)
+    {
+        $fs = new Filesystem();
+        if (!$fs->exists($outputDir)) {
+            $fs->mkdir($outputDir);
```

Comment:
ok I remove it
but when you launch the command, it fails... that's a strange sensation

maybe the doc is bad